lib/activity_model.dart

```diff
@@ -146,7 +158,7 @@ class ActivityModel extends Model {
     });
   }
 
-  void addImage(String imagePath) {
+  void addImage(String imagePath,Color c,BlendMode blnd) {
```

Comment:
You are taking 2 extra parameters: Color c,BlendMode blnd - but not doing anything with it